dist/module.cjs

```diff
@@ -0,0 +1,5 @@
+module.exports = function(...args) {
```

Comment:
Any reason why you push the `dist` folder Ideally this will be auto-generated and only available on NPM.

Reply:
You're correct, that was intentional because as requested [here](https://github.com/nuxt-community/device-module/pull/119#issuecomment-1167084698), people were trying to install it from GitHub. It wasn't included when I opened this PR, but 5 months passed and no one from this repository showed up so I temporarily included it.

Reply:
Ahh, got it 👍🏻
Yes, that makes sense - alternatively feel free to publish a temp package so people can use that while the PR is pending 😋